packages/core/content-manager/server/src/history/models/history-version.ts

```diff
@@ -0,0 +1,48 @@
+import type { Model } from '@strapi/database';
+import { HISTORY_VERSION_UID } from '../constants';
+
+const historyVersion: Model = {
+  uid: HISTORY_VERSION_UID,
+  tableName: 'strapi_history_versions',
+  singularName: 'history-version',
+  attributes: {
+    id: {
+      type: 'increments',
+    },
+    contentType: {
+      type: 'string',
+      column: { notNullable: true },
+    },
+    // documentId is a reserved attribute name
+    relatedDocumentId: {
+      type: 'string',
+      column: { notNullable: true },
+    },
+    locale: {
+      type: 'string',
+    },
+    status: {
+      type: 'enumeration',
+      enum: ['draft', 'published', 'modified'],
+    },
+    data: {
+      type: 'json',
+    },
+    schema: {
+      type: 'json',
+    },
+    createdAt: {
+      type: 'datetime',
+    },
```

Comment:
maybe not necessary!  but I believe you can add a default function like default: () => new Date()